app/views/students/registrations/_register.html.erb

```diff
@@ -17,16 +17,37 @@
   </select>
 </div>
 
-<div class="field">
+<!-- <div class="field">
   <%= f.label :hall %><br />
   <select name="student[hall]" class="form-control">
     <%= render 'sharedviews/halls' %>
   </select>
+</div> -->
+
+<hr>
+
+<h4>In which field you want to be Mentored in.</h4>
+
+<div class="well">
+  <div class="field">
+    <%= f.label :preference1 %>
+    <%= f.text_field :preference1 %>
+  </div>
+
+  <div class="field">
+    <%= f.label :preference2 %>
+    <%= f.text_field :preference2 %>
+  </div>
+
+  <div class="field">
+    <%= f.label :preference3 %>
+    <%= f.text_field :preference3 %>
+  </div>
 </div>
 
 <div class="field">
-  <%= f.label :joining_year %><br />
-  <%= f.text_field :joinyear %>
+  <%= f.label :current_academic_year %><br />
```

Comment:
@aniarya82 Actually, let the joining year field be there. We may need it later. And parsing it out of the roll number string can be unnecessarily tough. 

So, add both the joining year and the current academic year fields. 

And also, the current academic year field needs to be a drop down. Can you do that or should I do it after the code is merged?

Best Regards,

Siddharth Kannan,
Sophomore, Dept. of Mechanical Engg,
Indian Institute of Technology, Kharagpur.

> On 04-Mar-2015, at 13:39, Hiten Chowdhary <notifications@github.com> wrote:
> 
> In app/views/students/registrations/_register.html.erb:
> 
> >  </div>
> >  
> >  <div class="field">
> > -  <%= f.label :joining_year %><br />
> > -  <%= f.text_field :joinyear %>
> > +  <%= f.label :current_academic_year %><br />
> but we can know the join year from the roll number itself
> 
> —
> Reply to this email directly or view it on GitHub.
>